ibis/clickhouse/tests/test_select.py

```diff
@@ -292,6 +292,42 @@ def test_simple_joins(con, db, batting, awards_players,
     assert len(con.execute(expr))
 
 
+def test_multiple_joins(con, db, batting, awards_players, alltypes):
+    b, ap, a = batting, awards_players, alltypes
+
+    joined1 = (b.join(ap, ['date', 'lgID', 'yearID', 'playerID'])
+                .join(a, b.date == a.date))
+    joined2 = (b.join(ap, ['date', 'lgID', 'yearID', 'playerID'])
+                .join(a, ['date']))
+
+    # TODO fix aliasing
+    expected1 = """SELECT *
+FROM (
+  SELECT *
+  FROM ibis_testing.`batting` t0
+    ALL INNER JOIN ibis_testing.`awards_players` t1
+      USING `date`, `lgID`, `yearID`, `playerID`
+)
```

Comment:
Does this actually run? Most databases AFAIK require you to provide subqueries with aliases. Does clickhouse not have this requirement?

Reply:
Certain verions of Clickhouse actually when You alias a subquery (probably a bug).